README.md

```diff
@@ -29,16 +29,11 @@ If you need support for other Google APIs, check out the [Google .NET API Client
 
 In order to build the code in this repository, you will need
 to [install DNVM][dnvm]. You do not need DNVM to use the code,
-provided as NuGet packages.
-
-Most projects in this repository require [gRPC], which has a
-component written in native code. Using that component from DNX
-currently [requires a workaround][grpc-workaround]; on Windows please run `fix_dnx_grpc.bat`
-before running any gRPC code.
+which is provided as NuGet packages.
 
 ## Specifying a Project ID
 
-Most `google-cloud-dotnet` libraries require a project ID. If you don't remember yours (or haven't created a project yet), navigate to the [Google Developers Console][google-developers-console] to view your project ID (or create a new project and get the ID then). Once done, record the value and make sure to pass it as a parameter to the methods that require it.
+Most Google Cloud Libraries for .NET require a project ID. If you don't remember yours (or haven't created a project yet), navigate to the [Google Developers Console][google-developers-console] to view your project ID (or create a new project and get the ID then). Once done, record the value and make sure to pass it as a parameter to the methods that require it.
```

Comment:
"or create a new project and get the ID then" might read a bit better as "or create a new project and then get the ID"